src/lib/src/component/navigation/navigation.component.test.setup.ts

```diff
@@ -10,11 +10,34 @@ import { PersistenceService } from '../../service/persistence/persistence.servic
 import { TestExecutionService } from '../../service/execution/test.execution.service';
 import { Response, ResponseOptions } from '@angular/http';
 import { ElementState } from '../../common/element-state';
+import { IndicatorFieldSetup } from '../../common/markers/field';
 
 export const HTTP_STATUS_OK = 200;
 export const HTTP_STATUS_CREATED = 201;
 export const HTTP_STATUS_ERROR = 500;
 
+export const testEditorIndicatorFieldSetup: IndicatorFieldSetup = {
+  fields: [
+    {
+      condition: (element) => element && element.name.endsWith('.tcl'),
+      states: [{
+        condition: (marker) => marker.testStatus === ElementState.Running,
```

Comment:
`TestExecutionService` still resides in web-workspace-navigator, and is used by `NavigationComponent`. Both use `ElementState`, which is also a test-specific concept that should be moved to test-editor-web, so getting rid of it would require quite a lot of refactoring, which I am reluctant to do as part of this PR…